Monehin/pulse-backend
# Deployment

### Description
This is an attempt to deploy the backend app.

### How to test
To run your application locally: `yarn develop`

For production, check out the created review app. The deployment will automatically pick `yarn start` as the default command.

### Known issue
A database instance needs to be attached to review apps.
